src/main/java/org/launchcode/techjobs/oo/test/JobTest.java

```diff
@@ -1,12 +1,104 @@
 package org.launchcode.techjobs.oo.test;
 
+import org.junit.Before;
+import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
+import org.launchcode.techjobs.oo.*;
+
+import static org.junit.Assert.*;
 
 /**
  * Created by LaunchCode
  */
 @RunWith(JUnit4.class)
 public class JobTest {
 
+    Job job_one;
+    Job job_two;
+    Job job_complete;
+    Job job_incomplete;
+    Job job_blank;
+
+    @Before
+    public void createJobOne() {
+        job_one = new Job();
+    }
+
+    @Before
+    public void createJobTwo() {
+        job_two = new Job();
+    }
+
+    @Before
+    public void createCompleteJob() {
+        job_complete = new Job("Product tester", new Employer("ACME"), new Location("Desert"), new PositionType("Quality control"), new CoreCompetency("Persistence"));
+    }
+
+    @Before
+    public void createIncompleteJob() {
+        job_incomplete = new Job("Junior Dev", new Employer("Spredly"), new Location("Durham"), new PositionType("Developer"), new CoreCompetency(""));
+    }
+
+    @Before
+    public void createBlankJob() {
+        job_blank = new Job("", new Employer(""), new Location(""), new PositionType(""), new CoreCompetency(""));
+    }
+
+    @Test
+    public void testSettingJobID () {
```

Comment:
```suggestion
    public void testSettingJobId () {
```
It looks like the autograder explicitly wants this method name to have `Id` in regular case instead of uppercase. Changing this resolves a few of the tests that did not pass.